test/opengl/user_state_check.cpp

```diff
@@ -0,0 +1,82 @@
+#include <csetjmp>
+#include <unistd.h>
+
+#if defined(__APPLE__)
```

Comment:
Tests that include GL headers have been problematic across the various build bots. Please see rewrap_texture.cpp for an example that doesn't anger any build bots.